README.md

```diff
@@ -16,108 +15,53 @@ Its first use case will be for proving that blocks on the Unichain L2 were built
 ## System Flows
 
 1. Registering a TEE Device (also referred to as a block builder)
-    
```

Comment:
I changed the README while I was at it, since it hadn't been updated from the last few PR's-worth of changes.